test/AWS.Deploy.Orchestration.UnitTests/Utilities/CloudApplicationNameGeneratorTests.cs

```diff
@@ -106,13 +106,11 @@ public async Task SuggestsValidNameAndRespectsExistingApplications()
             recommendation.ShouldEqual(expectedRecommendation);
         }
 
-        [Fact]
-        public async Task SuggestsValidNameAndRespectsExistingApplications_ProjectWithNumber()
+        [Theory]
+        [InlineData("SuperTest1", "SuperTest2")]
```

Comment:
Can you add the following tests:

`SuperTest` -> `SuperTest` (There is no matching existing project)
`SuperTest` -> `SuperTest1` (There is just one matching existing project.

Reply:
done